### ISSUE TITLE
Kamstrup 401

### ISSUE THREAD

**Issue by steinerjakob** [golles/Home-Assistant-Sensor-MC66C#24]
### The request

Hi,
I just want to inform you that this integration works with the Kamstrup Multical 401.
Would be great if you could update this integration to work like the Kamstrup 401 integration, so you could change the settings via the home assistant ui.

Thanks for the great work!

**Comment by golles** [comment]
I agree, that would be nice. However, I don't have this meter anymore which makes it hard for me to verify any changes.

When looking at the current install base, 6 (for people who have analytics enabled), and 19 stars, it doesn't have a lot of community impact. 

The code for this integration is very outdated, I do try to keep a few things up to date but not really touching any code, afraid of breaking anything. 

It would maybe be better for anyone to fork my 403 integration and replace the meter specific code with the code from here.
